src/contracts/atlas/Atlas.sol

```diff
@@ -90,22 +89,21 @@ contract Atlas is Test, Factory {
 
     function execute(
         ProtocolCall calldata protocolCall,
-        UserMetaTx calldata userCall,
+        UserMetaTx calldata userMetaTx,
         SearcherCall[] calldata searcherCalls,
         address environment,
         bytes32 callChainHash
     ) external payable returns (uint256 accruedGasRebate) {
         // This is a self.call made externally so that it can be used with try/catch
         require(msg.sender == address(this), "ERR-F06 InvalidAccess");
-
+        {
+        require(callChainHash == CallVerification.getCallChainHash(protocolCall, userMetaTx, searcherCalls), "ERR-F07 InvalidSequence");
+        }
```

Comment:
Is there a specific reason for the {} curly braces wrapping the require? This is often used in Solidity to allocate a new stack frame ([scoping docs](https://docs.soliditylang.org/en/v0.8.11/control-structures.html#scoping-and-declarations)) but this seems to compile fine with no stack too deep errors when I remove the {}

Reply:
Good catch! Yes - in an earlier version of this PR I was hitting 'stack to deep,' hence the frame.

We can definitely remove it now.